dist/styles.css

```diff
@@ -0,0 +1,132 @@
+#app {
+  font-family: 'Roboto';
+}
+
+.fa-arrow-right {
+  margin-left: 10px;
+}
+
+.fa-arrow-right:hover {
+  margin-left: 15px;
+}
+
+.topImg {
+  display: flex;
+  justify-content: center;
+  width: 60%;
+  height: 50px;
+  margin-left: auto;
+  margin-right: auto;
+}
+
+.shop {
+  display: flex;
+  flex-wrap: wrap;
+  background-color: #f8ebe6;
+  padding: 2.5%;
+}
+
+.shopInfo {
+  font-size: 20px;
+  display: flex;
+  flex-direction: column;
+  margin-right: 5%;
+  margin-left: 10%;
+}

```

Comment:
big thumbs up on the flex box use

Reply:
Thanks! Yeah I spent alot of the day refactoring because it was even worse than it is now lol